src/static/css/commandline.css

```diff
@@ -139,7 +139,6 @@ input:focus {
 
 .url {
     text-decoration: var(--tridactyl-url-text-decoration);
-    padding-left: 7px;
```

Comment:
Not sure what this padding was for, but it was causing all sorts of spacing and alignment issues. If it's just to add separation between the title and content cells, maybe we should add padding to `td.content` instead?

Reply:
It was added during this PR https://github.com/tridactyl/tridactyl/pull/537 which was not the best documented, I have no idea what it was for. Remove it and if nothing seems to break we're probably fine :)